components/core/src/clp_s/clp-s.cpp

```diff
@@ -80,6 +81,17 @@ int main(int argc, char const* argv[]) {
             return 1;
         }
 
+        auto timestamp_dict = clp_s::ReaderUtils::read_timestamp_dictionary(archives_dir);
+        AddTimestampConditions add_timestamp_conditions(
+                timestamp_dict,
+                command_line_arguments.get_search_begin_ts(),
+                command_line_arguments.get_search_end_ts()
+        );
+        if (expr = add_timestamp_conditions.run(expr); std::dynamic_pointer_cast<EmptyExpr>(expr)) {
+            SPDLOG_ERROR("Query '{}' is logically false", query);
```

Comment:
This error is a little misleading since the `'{}'` doesn't mention anything about the timestamp conditions.

Reply:
Yeah, most of the errors in this section of the code aren't very clear. I can change this to an error that mentions whether the authoritative timestamp column exists so that the user knows they probably just missed the --timestamp-key argument at ingestion time.